### ISSUE TITLE
Unknown type name 'IndustriesManufacturingSettings' specified in package.xml in v7.88.4 and 7.89.1

### ISSUE THREAD

**Issue by MaciekLeks** [forcedotcom/cli#866]
### Summary

After upgrading version from 7.86.3 to either 7.88.4 or 7.89.1 we get 
```
Error  sdx_sourceDeploy_pkg_1614089165664/package.xml        Unknown type name 'IndustriesManufacturingSettings' specified in package.xml
```
while we run:
```
sfdx force:source:deploy -p force-app -u ALIAS -c -w 60 -l RunLocalTests
```

We had to downgarde to 7.86.3 cause we have nothing to do with that settings in our code.

### Step to reproduce

It could be hard to reproduce while it could be related to our force-app, but you can try to use the sfdx force:source:deploy instruction from the Summary.

**Comment by uip-robot-zz** [comment]
Git2Gus App is installed but the `.git2gus/config.json` doesn't have right values. You should add the required configuration.

**Comment by calvinle1** [comment]
Hi, our developers are also experiencing this issue with version 7.89.1.

Can you please confirm?

**Comment by MaciekLeks** [comment]
> Hi, our developers are also experiencing this issue with version 7.89.1.
> 
> Can you please confirm?

Yes, the same result. I've updated the title.

**Comment by calvinle1** [comment]
> > Hi, our developers are also experiencing this issue with version 7.89.1.
> > Can you please confirm?
> 
> Yes, the same result. I've updated the title.

Thanks

**Comment by calvinle1** [comment]
Hi, any resolution to this, other than downgrading the CLI? Thanks

**Comment by MaciekLeks** [comment]
I discovered what causes the "Unknown type name 'IndustriesManufacturingSettings' specified in package.xml" error in every sfdx-cli version except v7.86.3 in our project: File force-app/setup/clm/setttings/Content.settings-meta.xml needed for  Vlocity CLM. Empty project only with this file raises the error. The full project without it compile without the error. And file content is:
```
<?xml version="1.0" encoding="UTF-8"?>
<ContentSettings xmlns="http://soap.sforce.com/2006/04/metadata">
    <enableContent>true</enableContent>
    <enableContentAutoAssign>true</enableContentAutoAssign>
</ContentSettings>
```

**Comment by mshanemc** [comment]
I can't reproduce this.  I'm on v7.93.1, with the restDeploy=false.  The file you supplied is the only thing I added to the results of `project:create`.  Anything else required to get the error?

<img width="1343" alt="Screen Shot 2021-04-02 at 9 12 55 AM" src="https://user-images.githubusercontent.com/4261788/113423285-cf295480-9393-11eb-8f54-02119254e1d1.png">

**Comment by MaciekLeks** [comment]
> I can't reproduce this. I'm on v7.93.1, with the restDeploy=false. The file you supplied is the only thing I added to the results of `project:create`. Anything else required to get the error?

Have you installed vlocity cme before trying to sfdx deploy? We use (sfdx-project.json excerpt):
```
"dependencies": [
        {
          "subscriberPackageVersionId": "04t1J000001BRWt"
        }
```

**Comment by hnep12** [comment]
This issue is occurring for OmniChannel.settings deployment as well. Below is the code. 

<?xml version="1.0" encoding="UTF-8"?>
<OmniChannelSettings xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <enableOmniAutoLoginPrompt xsi:nil="true"/>
    <enableOmniChannel>true</enableOmniChannel>
    <enableOmniSecondaryRoutingPriority xsi:nil="true"/>
    <enableOmniSkillsRouting>true</enableOmniSkillsRouting>
</OmniChannelSettings>

**Comment by maggiben** [comment]
A fix has been put on toolbelt, users facing this issue can try the following:

1) upgrade to the latest sfdx-cli via npm or tarballs
2) clone the toolbelt repo
`git clone https://github.com/salesforcecli/toolbelt.git`
3) change directory
`cd toolbelt`
4) checkout branch with fix
`git checkout bm/W-9543092`
5) install plugin dependencies you're going to need yarn installed globally via npm `npm install -g yarn`
`yarn install`
6) link local plugin to sfdx-cli
`sfdx plugins:link .`
7) then try deploying as you would normally ie:
`sfdx force:source:deploy -p force-app -u ALIAS -c -w 60 -l RunLocalTests`

**Comment by madhavapakash** [comment]
@maggiben - please we do not find this repo at all git clone https://github.com/salesforcecli/toolbelt.git

**Comment by maggiben** [comment]
Please lease check that `layout` files in your org are not inside any other directory but `layouts` it's very important not to nest this type of data.

**Comment by maggiben** [comment]
Hi, is this issue still happening for you ?, please post below:

**Comment by tomekind92** [comment]
@maggiben the issue still exists for `sfdx force:source:push`

**Comment by ArkadiuszGrzesiak** [comment]
@maggiben I checked on sfdx-cli / 7.118.1 and 7.117.0 and after using `sfdx force:source:push` the IndustriesManufacturingSettings error still exists 
![IndustriesManufacturingSettings](https://user-images.githubusercontent.com/70268706/134458712-95029efb-6965-4c19-a376-d1b2230c3e4d.jpg)

**Comment by shetzel** [comment]
@ArkadiuszGrzesiak - what is the output of `sfdx plugins --core`?  I'm mostly interested in if there are any non-core plugins installed that could be causing a problem.  I can't reproduce this with a few recent CLI versions I tried.

From the error it looks like the name of that settings file might be incorrect which causes a bad manifest entry such as, `<members>IndustriesManufacturingSettings</members>`.  The name of the file in your project should be `IndustriesManufacturing.settings-meta.xml`.  If you set the `SFDX_MDAPI_TEMP_DIR` env var and run the push command again, then check the specified temp dir it should look like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
  <types>
    <name>Settings</name>
    <members>IndustriesManufacturing</members>
  </types>
  <version>52.0</version>
</Package>
```

**Comment by MaciekLeks** [comment]
Please reopen the issue. My teammates posted two last messages showing that the problem still exists. We do not use Industries Manufacturing at all.

**Comment by shetzel** [comment]
@MaciekLeks - this issue needs reproducible steps.

**Comment by tomekind92** [comment]
The issue still exists when I pushing the code do scratch/sandbox org. I've added the "IndustriesManufacturing.settings" into my deployment package and I have the below error:

| IndustriesManufacturing | Settings | "Not available for deploy for this organization" |

**Comment by jacekpreuhs** [comment]
Hi @shetzel ,
I've made some additional tests:
1) configured SFDX_MDAPI_TEMP_DIR variable as you mentioned
2) used sfdx force:source:push -f -u DEV command to test deployment and got error:
   Unknown type name 'IndustriesManufacturingSettings' specified in package.xml
3) IndustriesManufacturingSettings folder was created in SFDX_MDAPI_TEMP_DIR folder
4) package.xml was created in folder with following xml tags:
  <types>
    <name>IndustriesManufacturingSettings</name>
    <members>Content</members>
  </types>

Please find attached screenshots. 
![image](https://user-images.githubusercontent.com/41343127/138257189-11dded87-b66b-4945-8e90-84dfffb0a9c0.png)
![image](https://user-images.githubusercontent.com/41343127/138257265-48acaa6c-b443-4d5d-9688-f520bba3aa49.png)
![image](https://user-images.githubusercontent.com/41343127/138257286-f04131ff-1f4b-4077-a506-98824d8337d1.png)
![image](https://user-images.githubusercontent.com/41343127/138257346-2f289892-5e1d-4f00-b4d7-39f33b006991.png)
![image](https://user-images.githubusercontent.com/41343127/138257603-791e878e-bfbc-4d0c-85f9-f7ab269fc182.png)

cc: @MaciekLeks @ArkadiuszGrzesiak

**Comment by WillieRuemmele** [comment]
Hi @jacekpreuhs 

a few questions:
what version of the cli and the plugins are you using `sfdx --version --verbose`?
have you enabled this type in the org? In `Setup` search for `Manufacturing` and enable it there, or in the `project-scratch-def.json` add `"INDUSTRIESMFGACCOUNTFORECAST"` to the `features` array

Cristian has created [this repo](https://github.com/cristiand391/industry-manufacturing-settings) to try and replicate the issue. Using this repo
- create an org
- enable the type in setup
- retrieve it locally with `sfdx force:source:retrieve -x package.xml`
- run `sfdx force:source:push -f` and I see the following output
``` ➜  push -f
*** Deploying with SOAP ***
Job ID | 0Af8A00001PTykiSAD
SOURCE PROGRESS | ████████████████████████████████████████ | 1/1 Components
=== Pushed Source
STATE  FULL NAME                TYPE      PROJECT PATH
─────  ───────────────────────  ────────  ─────────────────────────────────────────────────────────────────────────
Add    IndustriesManufacturing  Settings  force-app/main/default/settings/IndustriesManufacturing.settings-meta.xml
```

this was done using `7.123.1` of the sfdx-cli

**Comment by MaciekLeks** [comment]
Please reopen the issue.

**Comment by WillieRuemmele** [comment]
Hi @MaciekLeks we'll reopen this issue once we are able to reproduce it. I've just asked for some information you, or anyone else on this thread could provide. I've provided a repository to try and reproduce it but was unable to with the current version of the CLI. Can you help me reproduce it?

**Comment by jacekpreuhs** [comment]
Hi @WillieRuemmele ,

1) below you can find result of `sfdx --version --verbose`
![image](https://user-images.githubusercontent.com/41343127/139389806-d1591b15-7b11-492b-988f-df80432f135f.png)

2) in our org (scratch or Production) we do not have Manufacturing feature enabled and we do not planning to use this feature. As I know this relates to Manufacturing Cloud: https://www.salesforce.com/ap/editions-pricing/manufacturing-cloud/
![image](https://user-images.githubusercontent.com/41343127/139397771-615c5e99-cca4-4c2a-80e0-ba6be004950f.png)


3) I do not understand why we need to turn on Manufacturing feature in order to resolve deployment errors when we don't want to use it at all. Se previous comments from @MaciekLeks on 26th March. He explained that deployment error occurs when following content settings is used in repo:
```
<?xml version="1.0" encoding="UTF-8"?>
<ContentSettings xmlns="http://soap.sforce.com/2006/04/metadata">
    <enableContent>true</enableContent>
    <enableContentAutoAssign>true</enableContentAutoAssign>
</ContentSettings>
```
I do not see here any feature related to Manufacturing. Could you please explain why Manufacturing is required for this content settings configuration?

Thanks,
Jacek

**Comment by WillieRuemmele** [comment]
Hi @jacekpreuhs 

thanks for the info. I agree - you shouldn't have to enable that setting if you're not using it. 

can you please walk us through how to recreate this bug, from how you create the metadata type, what commands are involved, and any steps in the org's UI that're required. 

Thanks

**Comment by pawel-id** [comment]
Why are we trying to discover the wheel again? This issue was happening to `force:source:deploy` and it was fixed in 7.116.2 see https://github.com/forcedotcom/cli/blob/main/releasenotes/sfdx/README.md#71162-sep-2-2021 there is even a reference to the issue we are discussing right now:

> FIX: The force:source:deploy command correctly deploys the IndustriesManufacturingSettings metadata type. Previously it returned the error Unknown type name 'IndustriesManufacturingSettings' specified in package.xml.

Then we discovered that it is happening also to `force:source:push`. Why not check and potentially reuse the solution applied with that fix?
